Assets/Scripts/UI/PauseMenuUI.cs

```diff
@@ -25,23 +27,30 @@ private void Update()
         {
             if(gameIsPaused)
             {
-                Resume();
+                Resume(prevCursorLockMode);
             }
             else
             {
                 Pause();    
             }
         }
     }
-    public void Resume()
+    public void Resume(CursorLockMode cursorLockMode)
```

Comment:
Adding a parameter to this broke the pause menu prefab (the resume button)

Reply:
Fixed! I could not see anything in the pause menu UI prefab was testing it, I had assumed that it was not implemented yet.